### ISSUE TITLE
Support for multiple lightbox instances.

### ISSUE THREAD

**Issue by ABackw** [rqrauhvmra/Tobii#103]
Old rqrauhvmra tobi-library worked with multiple instances. Why the support has been dropped?

Would it be possible to add a guid to class names (or as a user defined classname as input parameter) instead of hard coded names so the lightbox would work if there are multiple instances of it.

**Comment by midzer** [comment]
@ABackw What is the benefit of having multiple tobii instances? Whats a use case to allow them?

**Comment by kallaviktor** [comment]
@midzer I have multiple gallerys on my page, and I would like to only show a set of images when lightbox opened.

**Comment by viliusle** [comment]
You can use different groups, so even on 1 lightbox instance, users will see 2 galleries.

**Comment by kallaviktor** [comment]
Thank you, I have not noted this group option.

**Comment by midzer** [comment]
Is this issue solved for you, @ABackw @kallaviktor?

**Comment by ABackw** [comment]
> Is this issue solved for you, @ABackw @kallaviktor?

Yes, I didn't come up with any benefits of multiple instances other than easier migration from the old version. That's not good enough reason. I'm fine with current state of things.

**Comment by scheibome** [comment]
We have another reason why we would love to have support for multiple instances. In one web app (and therefore installed the requirement once via npm) we have multiple sites/themes with different requirements for lightboxes (e.g. One site wants a very “light” lightbox with a different styling/theme than the others. What would be the usage of “themes” without this feature?

**Comment by midzer** [comment]
Done via https://github.com/midzer/tobii/releases/tag/v2.8.1
